app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -362,14 +361,6 @@ def add_proofing_costs(results)
       end
     end
 
-    def track_aamva
-      return unless IdentityConfig.store.state_tracking_enabled
-      doc_auth_log = DocAuthLog.find_by(user_id: current_user.id)
-      return unless doc_auth_log
-      doc_auth_log.aamva = true
```

Comment:
we could prob add `aamva` as an ignored column to the table if we wanted?

Reply:
@matthinz Is the plan to follow-up with a pull request to drop the column in a separate deploy? I also noticed on Friday that [the other columns in `ignored_column` were meant to be cleaned up last year](https://github.com/18F/identity-idp/pull/8272) but never were.